cmd/security-proxy-setup/res/docker/configuration.toml

```diff
@@ -1,96 +0,0 @@
-#################################################################################
-# Copyright 2019 Dell Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
-# in compliance with the License. You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software distributed under the License
-# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
-# or implied. See the License for the specific language governing permissions and limitations under
-# the License.
-#
-# @author: Tingyu Zeng, Dell
-#################################################################################
-
-# This is a TOML config file for edgexsecurity service.
-
-[Writable]
-LogLevel = "DEBUG"
-RequestTimeout = 10
-
-[Logging]
-EnableRemote = false
-File = "./logs/security-proxy-setup.log"
-
-[KongURL]
-Server = "kong"
-AdminPort = 8001
-AdminPortSSL = 8444
-ApplicationPort = 8000
-ApplicationPortSSL = 8443
-
-[KongAuth]
-Name = "jwt"
-TokenTTL = 0
-Resource = "coredata"
-OutputPath = "accessToken.json"
-
-[KongACL]
-Name = "acl"
-WhiteList = "admin"
-
-[SecretService]
-Server = "edgex-vault"
-Port = 8200
-HealthCheckPath = "v1/sys/health"
-CertPath = "/v1/secret/edgex/edgex-security-proxy-setup/"
```

Comment:
OK, I figured out why security-proxy-setup is failing. Note the leading "/" in `CertPath`, @tingyuz added this in a recent PR, however it appears he didn't make the same change in the non-docker configuration.toml.

The value of `HealthCheckPath` looks suspect as well (i.e. no leading "/"), so we should probably check on this as well. It'd be terrible if some of our configuration values which are keys in vault or console require the leading "/" and some don't.